temba/msgs/models.py

```diff
@@ -960,13 +963,18 @@ def mark_sent(cls, r, msg, status, external_id=None):
             Msg.objects.filter(id=msg.id).update(status=status, sent_on=msg.sent_on)
 
     def as_json(self):
-        return dict(direction=self.direction,
+        data = dict(direction=self.direction,
```

Comment:
Let's move away from using `data` as a variable, it isn't descriptive at all. `msg_json` would be better.